src/vocab_encoder.cpp

```diff
@@ -45,11 +47,11 @@ bool VocabEncoder::evaluate(ov::TensorVector& outputs, const ov::TensorVector& i
         auto vocab_chars  = inputs[5].data<const uint8_t>();
 
         auto vocab_values = inputs[6].data<const int32_t>();
-        auto vocab_size = inputs[6].get_size();
+        const auto vocab_size = inputs[6].get_size();
 
-        m_vocab = std::make_shared<std::map<std::vector<unsigned char>, int32_t>>();
+        m_vocab = std::make_shared<absl::flat_hash_map<std::string, int32_t>>();
         for (size_t i = 0; i < vocab_size; ++i) {
```

Comment:
can we parallelize it?

Reply:
It could speed up the first inference, will do it later.